typed-racket-lib/typed-racket/rep/type-mask.rkt

```diff
@@ -4,7 +4,7 @@
 ;; Type Masks
```

Comment:
Is a `Self` really something that is never described by any other type mask? I.e., can't a `Self` be a struct?

I would have assumed `Self` would have the same type mask as structs (or similar).